packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md

```diff
@@ -40,13 +36,13 @@ Identity engine provides:
 
 ## New concepts and terminology
 
-Identity engine introduces new concepts and terminology.
+Identity Engine introduces new concepts and terminology.
 
 * Assurance
 
   Assurance is the degree of confidence that an end user signing in to an application is the same end user who previously signed in to the application, and the use of one or more authenticators (see below) and its characteristics determine an assurance level. For example, an end user who authenticates with a knowledge factor and a possession factor has a higher assurance level than one who can only authenticate with one factor.
```

Comment:
(see below) could be considered a little vague, but a link is pretty useless. Maybe (next bullet)? Just a suggestion